src/main/java/next/mapper/UserRowMapper.java

```diff
@@ -0,0 +1,47 @@
+package next.mapper;
+
+import core.jdbc.RowMapper;
+import next.model.User;
+import support.exception.ParameterClassNotFoundException;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class UserRowMapper implements RowMapper<User> {
+
+    @Override
+    public User mapRow(ResultSet resultSet) {
+        try {
+            return getObjectFromRow(resultSet);
+        } catch (Exception exception) {
+            throw new RuntimeException(exception);
+        }
```

Comment:
피드백을 반영해주셨네요. :)

```suggestion
        return new User(rs.getString("userId"),
                rs.getString("password"),
                rs.getString("name"),
                rs.getString("email"));
```
UserRowMapper는 User를 반환하게 만들어졌으니 Reflection을 사용하지 않고 이렇게 만들어보면 어떨까요?

Reply:
반영하였습니다~